tests/client/feedback/training/test_trainer.py

```diff
@@ -0,0 +1,97 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import TYPE_CHECKING, List, Union
+
+import pytest
+
+if TYPE_CHECKING:
+    from argilla.client.feedback.schemas import AllowedFieldTypes, AllowedQuestionTypes
+
+import shutil
+import sys
+from pathlib import Path
+
+from argilla.client.feedback.dataset import FeedbackDataset
+from argilla.client.feedback.schemas import (
+    FeedbackRecord,
+    LabelQuestion,
+    MultiLabelQuestion,
+)
+from argilla.client.feedback.training import ArgillaTrainer
+from argilla.client.feedback.training.schemas import (
+    TrainingTaskMapping,
+)
+from argilla.client.feedback.unification import LabelQuestionUnification
+from argilla.client.models import Framework
+
+__OUTPUT_DIR__ = "tmp"
+
+
+@pytest.mark.parametrize(
+    "framework",
+    [
+        Framework("spacy"),
+        Framework("transformers"),
+        Framework("spark-nlp"),
+        # Framework("autotrain"),
+        Framework("span_marker"),
+        Framework("setfit"),
+        Framework("peft"),
+        # Framework("openai"),
```

Comment:
If we're not running the tests for those, just remove them instead of leaving the commented line there IMO